### PR TITLE
Ramp up amp-carousel Scroll Snap experiment to 1% of prod

### PR DESCRIPTION
**Pull request by nainar**

### CODE REVIEW

**Comment by nainar** [comment]
Do you think rolling out to 1% of prod would be safer? From my understanding we test canary config on 1% of prod anyway so I thought the next logical step would be 10% of prod pages. Let me know if you want to ramp it up more slowly @cathyxz

**Comment by cathyxz** [comment]
Depends on whether you want to monitor canary logs, or rather how long you would need to monitor canary logs. If you need a week for that, then just do 1%.

File: build-system/global-configs/prod-config.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
   "amp-consent": 1,
   "amp-img-native-srcset": 1,
   "amp-story-v1": 1,
+  "amp-carousel-scroll-snap": 10,
```
Comment (cathyxz) [inline build-system/global-configs/prod-config.json]: Shouldn't this be 0.1?

Comment (nainar) [reply build-system/global-configs/prod-config.json]: Oh lol  - then I turned the experiment on for 100% of canary? Just confirmed with Raghu that this should be 0.1 indeed. Changed this to `0.01` to signify a 1% prod promotion.

**Comment by nainar** [comment]
@cathyxz changed to a 1% prod experiment.

**Pull request merged**